src/uct/ib/base/ib_md.c

```diff
@@ -129,6 +129,11 @@ static ucs_config_field_t uct_ib_md_config_table[] = {
    "Prefer nearest device to cpu when selecting a device from NET_DEVICES list.\n",
    ucs_offsetof(uct_ib_md_config_t, ext.prefer_nearest_device), UCS_CONFIG_TYPE_BOOL},
 
+  {"CONTIG_PAGES", "n",
+   "Enable allocation with contiguous pages. Warning: enabling this option may\n"
+   "cause stack smashing.\n",
```

Comment:
unfortunately, contig pages with fixed addr from mmap does not work well when the app is multi-threaded. Another thread might get from mmap() the same virtual address that is passed to contig allocator, since libmlx5 does several attempts of mmap/munmap with different "block size".